src/index.test.ts

```diff
@@ -1,12 +1,16 @@
-import { describe, test, before } from "node:test";
+import { before, describe, test } from "node:test";
+import { getKeypairFromEnvironment, getKeypairFromFile } from "./index";
+
+import { Keypair } from "@solana/web3.js";
 import assert from "node:assert/strict";
-import { promisify } from "util";
+import base58 from "bs58";
+import dotenv from "dotenv";
 import { exec as execForOldPeople } from "child_process";
-const exec = promisify(execForOldPeople);
-import { Keypair } from "@solana/web3.js";
-import { getKeypairFromFile, getKeypairFromEnvironment } from "./index";
+import { promisify } from "util";
+import { unlinkSync } from "node:fs";
```

Comment:
Just use the FS promises versions 

https://nodejs.org/api/fs.html#fspromisesunlinkpath

https://nodejs.org/api/fs.html#fspromisesappendfilepath-data-options